html-relative-links/projects/projects.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Projects</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href='../index.html'>Home Page</a></li>
+    <li><a href='../about-me/about-me.html'>About Me</a></li>
+    <li><a href='projects.html'>Projects</a></li>
+    <li><a href='../contact-us/contact-us.html'>Contact Us</a></li>
+  </ul>
+  <p>Hello Projects!</p>
+  <img src="https://pbs.twimg.com/media/EwxVoP5W8Ac4cVV.jpg"
```

Comment:
Please use a relative path to the images included in the starter files for this exercise.